### DISCUSSION TITLE
Make Publish buttons mistap-proof for smartphones (editor UI)

### DISCUSSION THREAD

**Discussion by yheuhtozr** [forem/forem#15918] [Feature Requests]
My instance has a considerable percentage of mobile authors, and they report that the current editor UI is susceptible to accidental premature submission because:

1. Buttons are on the bottom of the screen where you easily mistap trying to select a predicted word (see below)
2. "Publish" and "Save draft" buttons are next to each other that you can easily mistap another on a small screen
3. Tapping "Publish" button immediately takes effect with no buffer

They also suggest whether we can move action buttons to the top side like Twitter's mobile UI.

#### About buttons and predictive text

Android phones may be shipped with built-in keyboards that have foldable prediction field, which looks as follows depending on input state (screenshots are from my only Android tablet; please refer to [the original Japanese post](https://migdal.jp/azumanyan/migdal-%E3%82%92%E4%BD%BF%E3%81%B2%E5%A7%8B%E3%82%81%E3%82%8B%E3%81%A8%E3%81%A9%E3%81%86%E3%81%97%E3%81%A6%E3%82%82%E9%80%94%E4%B8%AD%E3%81%A7%E5%85%AC%E9%96%8B%E3%81%97%E3%81%A6%E3%81%97%E3%81%BE%E3%81%B5%E8%A9%B1-81j) for actual handheld device screens). You may completely turn off prediction in English keyboard, but Japanese (and Chinese) keyboard relies on phonetic dictionary lookup which display results via the same function.

not inputting (prediction off)|inputting (prediction on)
:-:|:-:
![prediction off](https://user-images.githubusercontent.com/84892012/147874397-def939c3-55d6-4d6f-84ad-3a4d26357fd4.png)|![prediction on](https://user-images.githubusercontent.com/84892012/147874402-b4a0f0cc-0506-4a06-8bab-4b3c56644911.png)

**Comment by abdemiller87** [comment]
Update your WordPress to the latest version 9.0 which was released recently. My client is also facing this error while posting in their [social spy whatsapp](https://socialspy.net/). The New version is awesome must try it.
